tests/devpool-issue-fork-handler.test.ts

```diff
@@ -0,0 +1,356 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { drop } from "@mswjs/data";
+import { setupServer } from "msw/node";
+import { db } from "../mocks/db";
+import { handlers } from "../mocks/handlers";
+import issueDevpoolTemplate from "../mocks/issue-devpool-template.json";
+import issueTemplate from "../mocks/issue-template.json";
+import { GitHubIssue } from "../src/directory/directory";
+import { updateDirectoryIssue } from "../src/directory/update-issue";
+
+const DEVPOOL_OWNER_NAME = "ubiquity";
+const DEVPOOL_REPO_NAME = "devpool-directory";
+
+const server = setupServer(...handlers);
+
+jest.mock("../src/directory/directory", () => ({
+  ...jest.requireActual("../src/directory/directory"),
+  DEVPOOL_OWNER_NAME: "not-ubiquity",
+}));
+
+beforeAll(() => server.listen());
+afterEach(() => {
+  server.resetHandlers();
+  drop(db);
+});
+afterAll(() => {
+  server.close();
+  jest.unmock("../src/directory/directory");
+});
+
+function createIssues(devpoolIssue: GitHubIssue, projectIssue: GitHubIssue) {
+  db.issue.create(devpoolIssue);
+  db.issue.create(projectIssue);
+
+  return db.issue.findFirst({
+    where: {
+      id: {
+        equals: devpoolIssue.id,
+      },
+    },
+  }) as GitHubIssue;
+}
+
+describe("handleForkedDevPoolIssue", () => {
+  const logSpy = jest.spyOn(console, "log").mockImplementation();
+
+  beforeEach(() => {
+    logSpy.mockClear();
+  });
+
+  describe("Forked Devpool", () => {
+    beforeEach(() => {
+      db.repo.create({
+        id: 1,
+        owner: "not-ubiquity",
+        name: "test-repo",
+        html_url: `https://github.com/not-ubiquity/test-repo`,
+      });
+      db.repo.create({
+        id: 2,
+        owner: DEVPOOL_OWNER_NAME,
+        name: DEVPOOL_REPO_NAME,
+        html_url: `https://github.com/${DEVPOOL_OWNER_NAME}/${DEVPOOL_REPO_NAME}`,
+      });
+    });
+
+    test("updates issue title in devpool when project issue title changes in forked repo", async () => {
+      const devpoolIssue = {
+        ...issueDevpoolTemplate,
+        id: 1,
+        title: "Original Title",
+        owner: "not-ubiquity",
+      } as GitHubIssue;
```

Comment:
I think we should add new template for forked devpool. Replacing just owner isn't enough. `issueDevpoolTemplate` has 

```
"html_url": "https://github.com/ubiquity/devpool-directory/issues/1",
"repository_url": "https://github.com/ubiquity/devpool-directory",
```

But here in this test suite, the devpool is not `ubiquity/devpool-directory` but it is a fork `not-ubiquity/devpool-directory`

```
"html_url": "https://github.com/not-ubiquity/devpool-directory/issues/1",
"repository_url": "https://github.com/not-ubiquity/devpool-directory",
```

so add a copy of `issueDevpoolTemplate` with correct urls with `issueDevpoolForkTemplate` and use it here.
This should help in making test purpose clearer.

Also, let's make rename `devpoolIssue` to `forkedDevpoolIssue` if it is fork.